Applications/Spire/Include/Spire/Ui/ScalarWidget.hpp

```diff
@@ -0,0 +1,60 @@
+#ifndef SPIRE_SCALAR_WIDGET_HPP
+#define SPIRE_SCALAR_WIDGET_HPP
+#include <QWidget>
+#include "Spire/Spire/Scalar.hpp"
+
+namespace Spire {
+
+  //! Provides a common widget with an interface for getting and setting a
```

Comment:
//! Wraps a QWidget used to edit a scalar type T into a QWidget that edits a Scalar.